common/lib/api-client/request-headers.js

```diff
@@ -2,10 +2,12 @@ const uuid = require('uuid')
 
 const {
   API: { VERSION },
+  APP_VERSION,
 } = require('../../../config')
 
 module.exports = (format = 'application/vnd.api+json') => {
   return {
+    'User-Agent': `hmpps-book-secure-move/${APP_VERSION}`,
```

Comment:
Do we want to append `-fronted` to be more explicit: `hmpps-book-secure-move-frontend/${APP_VERSION}`?

Reply:
Done

Reply:
@teneightfive Good idea, that makes it more obvious it is from our `frontend` client